arm-storsimple8000series/2017-06-01/swagger/storsimple.json

```diff
@@ -906,24 +999,80 @@
           }
         ],
         "responses": {
-          "202": {
-            "description": "Accepted the request to deactivate the device."
+          "200": {
+            "description": "The Alert settings.",
+            "schema": {
+              "$ref": "#/definitions/AlertSettings"
+            }
+          }
+        },
+        "deprecated": false
+      },
+      "put": {
+        "tags": [
+          "DeviceSettings"
+        ],
+        "description": "Creates or updates the alert settings of the specified device.",
+        "operationId": "DeviceSettings_CreateOrUpdateAlertSettings",
+        "consumes": [
+          "application/json"
+        ],
+        "produces": [
+          "application/json"
+        ],
+        "parameters": [
+          {
+            "name": "deviceName",
```

Comment:
Currently, for private preview of our service, we are using these APIs on portal through signed-dlls. These signed dlls are generated using this same swagger spec and the auto-rest. If the order is changed, it will affect all our APIs. Thus, at all places of use of these APIs we will need to change the order of parameters passed(portal code), needing new round of test-passes(all scenarios already covered earlier). 

And hence would prefer keeping them in current order.